packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/dashboard/manage-nsx-edges/dedicatedCloud-datacenter-manage-nsx.module.js

```diff
@@ -7,6 +7,10 @@ const moduleName = 'ovhManagerDedicatedCloudDatacenterManageNsxEdges';
 
 angular
   .module(moduleName, [datacenterManageNsxEdgesComponent])
+  .constant(
+    'NSX_TRACKING_PREFIX',
+    'dedicated::dedicatedCloud::details::datacenter::details::dashboard::nsx',
+  )
```

Comment:
**suggestion:** Better to define as a JS constant instead of AngularJS constant.